scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -1123,27 +1123,29 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
         <tr class="table-secondary">
           <th>Variant type</th>
           <th>Estimated size</th>
+          <th>Reference size</th>
           <th>Coordinates</th>
-          <th>Cytoband</th>
           <th>Gene panels</th>
         </tr>
       </thead>
       <tbody>
-        <tr>
+        {% if variant.str_status == 'full_mutation' %}
```

Comment:
It seems rather unlikely someone would mark a variant that has no status or reference size, but I guess they are not exactly mandatory. Since we are so careful with checking existence of e.g. inheritance mode perhaps better do.